examples/docking-protein-homotrimer/docking-protein-homotrimer.cfg

```diff
@@ -65,7 +65,7 @@ c3sym_seg3_1="C"
 
 
 #[caprieval]
```

Comment:
you need to uncomment also the header `[caprieval]`

Reply:
Hum - that file should not have been part of that pull request :-)  But I will correct it since it does fix the issue (partially) of caprieval on homomultimers

Reply:
`caprieval` is not fully compatible with multibody see #164

Reply:
Yes - but by making the modified a.a. HETATM it does work (at least we can do some evaluations of the performance -  but I don't know which interface / l-RMSD are being calculated)